src/commands/importExportEntities/entities/webhooks.ts

```diff
@@ -0,0 +1,44 @@
+import { WebhookContracts } from "@kontent-ai/management-sdk";
+import chalk from "chalk";
+
+import { logInfo } from "../../../log.js";
+import { serially } from "../../../utils/requests.js";
+import { FixReferences } from "../../../utils/types.js";
+import { EntityDefinition } from "../entityDefinition.js";
+
+type Webhook = FixReferences<WebhookContracts.IWebhookContract>;
+
+export const webhookEntity: EntityDefinition<ReadonlyArray<Webhook>> = {
+  name: "webhooks",
+  fetchEntities: client => client.listWebhooks().toPromise().then(res => res.rawData as ReadonlyArray<Webhook>),
+  serializeEntities: webhooks => JSON.stringify(webhooks),
+  deserializeEntities: JSON.parse,
+  importEntities: async (client, entities, context, logOptions) => {
+    await serially(entities.map(webhook => () => {
+      logInfo(logOptions, "verbose", `Importing: webhook ${webhook.id} (${chalk.yellow(webhook.name)})`);
+
+      return client
+        .addWebhook()
+        .withData({
+          name: webhook.name,
+          secret: webhook.secret,
+          url: webhook.url,
```

Comment:
This won't be so easy -> half of the the params are missing  -> you also need to transform references in events from id to codenames

Reply:
added transformation logic and missing params